src/ac-power/ac-power.c

```diff
@@ -12,21 +12,19 @@ static void help(void) {
                "Report whether we are connected to an external power source.\n\n"
                "  -h --help             Show this help\n"
                "     --version          Show package version\n"
-               "  -v --verbose          Show state as text\n"
-               , program_invocation_short_name);
+               "  -v --verbose          Show state as text\n",
```

Comment:
I prefer original version. It is better to minimize diff when we add a new option.